bundles/org.openhab.core.io.rest.voice/src/main/java/org/openhab/core/io/rest/voice/internal/VoiceResource.java

```diff
@@ -168,17 +187,32 @@ public Response interpret(
             @HeaderParam(HttpHeaders.ACCEPT_LANGUAGE) @Parameter(description = "language") @Nullable String language,
             @Parameter(description = "text to interpret", required = true) String text) {
         final Locale locale = localeService.getLocale(language);
-        HumanLanguageInterpreter hli = voiceManager.getHLI();
+        Collection<HumanLanguageInterpreter> hli = voiceManager.getDefaultHLIs();
```

Comment:
Please do not update this REST API. Its description is clear: "Sends a text to the default human language interpreter".
There is only one default HLI.